docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+SHARED_DIR="/var/lib/goofys"
+
+trap "umount ${SHARED_DIR}" SIGKILL SIGTERM SIGHUP SIGINT EXIT
+
+mount --make-shared $SHARED_DIR
+OPTS=" -o allow_other -f"
+
+if ! test -z "$GOOFYS_GID"; then
+  groupadd -g $GOOFYS_GID goofys
```

Comment:
Ah right, that's the reason why I pass UID/GID separately: users need to be created in the container, for UID/GID mapping reasons.